chart-app/src/components/ChartApp.vue

```diff
@@ -4,35 +4,46 @@
 
 <!--    This div is the container for the chart component-->
     <div class="chart-container">
-      <div class="chart-placeholder">
-
-      </div>
+      <!-- Using Chart.js in a Vue.js Application-->
+      <!-- Step 2. Modify App.vue to use the new Chart component-->
+      <SampleChart/>
     </div>
 
 <!--    This div is the container for the buttons-->
-    <div class="button-container">
-      <div class="dropdown-placeholder">
-
+    <div class="controls-container">
+
+      <div class="dropdown-container">
+        <select name="charts" id="chart-select">
+          <option value="">-- Please select a chart --</option>
+          <option value="chart1">Chart 1</option>
+          <option value="chart2">Chart 2</option>
+          <option value="chart3">Chart 3</option>
```

Comment:
```suggestion
          <option v-for="(chart, name) in charts" :key="name" :value="name">{{chart.label}}</option>
```

The v-for will turn 1 line into 3 lines. In this case you'd have a variable in your component like so:
```
data() {
   return {
     charts: {
        chart1: {label: 'Chart 1'},
        ...
```

It's also good practice to give each line a unique key